.gitignore

```diff
@@ -14,3 +14,4 @@
 /vendor/
 /nbproject/private/
 phpunit.xml
+/coverage/
```

Comment:
OK, think I've just ran phpunit directly from directory :)
Thanks for the advice anyways!